test/e2e/pkg/testnet.go

```diff
@@ -316,12 +321,6 @@ func (t Testnet) Validate() error {
 	if len(t.Nodes) == 0 {
 		return errors.New("network has no nodes")
 	}
-	if t.MaxInboundConnections < 0 {
-		return errors.New("MaxInboundConnections must not be negative")
-	}
-	if t.MaxOutboundConnections < 0 {
-		return errors.New("MaxOutboundConnections must not be negative")
-	}
```

Comment:
I added them to experiment with different network typologies i.e. without this, everyone eventually connects with everyone because the network is so small.

Reply:
cool, added back in I added them to experiment with